scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -76,7 +76,7 @@ function doInstallEclipsePlugin() {
   local pluginDependencies
   pluginDependencies=$(doCheckPluginsInstalled "${plugin_id}")
   local debugInfo
-  debugInfo=$(doInvokeEclipse org.eclipse.equinox.p2.director -repository "${plugin_url}" -installIU "${plugin_id}")
+  debugInfo=$(doInvokeEclipse org.eclipse.equinox.p2.director -repository "${plugin_url}" -installIU "${plugin_id}" -configuration "${DEVON_IDE_HOME}/software/eclipse-plugins/configuration/")
```

Comment:
you have already added `-configuration` to `doInvokeEclipse` so here you should not add it twice.